diagnostic.md

```diff
@@ -29,50 +29,58 @@ Why don't we use views in our interpretation of the MVC pattern?
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create Read Update Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+controllers/views
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+Create, Delete, Insert, Edit, Index
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails new <app_name>
 ```
 
 What is the command to start an instance of a rails server?
 
 ```bash
-// your response here
+bin/rails server
 ```
 
 What are the commands to drop, create, migrate and seed a database from the command
 line? (5 bullet points)
 
 ```bash
-// your response here
+-rake db:drop
+-rake db:create
+-rake db:migrate
+-rake db:seed
+
+# 5 bullet points? hmm?
+
 ```
 
 What is the command to scaffold a pet with a name and age attributes (hint:
 Also think of the data types for each attribute)?
 
 ```bash
-// your response here
+create Pet name:string, age:integer
```

Comment:
Looking for something like this: `rails-api g scaffold pet name:string age:integer`